app/helpers/inputs_helper.rb

```diff
@@ -1,10 +1,12 @@
 module InputsHelper
 
-  def text_input(name:, value:, label: nil, disabled: false, help: nil, error_message: nil)
-    render Input::TextInputComponent.new(label: input_label(label, name), name: name, value: value,
+  def text_input(name:, value:nil, label: nil, disabled: false, help: nil, error_message: nil, placeholder: nil, data: nil)
+    render Input::TextInputComponent.new(label: label ? label : input_label(label, name), name: name, value: value,
```

Comment:
`label ? label : input_label(label, name)` will break the expected behavior, in your case just call text_input with `labe: ''`